Template2/Template2/Stack.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Stack.swift
+//  Template2
+//
+//  Created by Дмитрий Логвинов on 06.02.2024.
+//
+
+// Stack.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+final class Stack<Element> {
+    private var storage: [Element] = []
+
+    func push(_ element: Element) {
+        storage.append(element)
+    }
+
+    func pop() -> Element? {
+        storage.popLast()
+    }
+
+    func peek() -> Element? {
```

Comment:
Вместо данной функции использовать вычисляемое свойство
```suggestion
    func peek() -> Element? {
    storage.last
```